libs/utils/src/lib/common.ts

```diff
@@ -7,7 +7,7 @@ export const print = (str?: string) => {
 export const printLocaleValue = (locale: LocaleCodes, value: any) => {
     if (!value) return;
     if (typeof value === 'string') return value;
-    return value[locale] || value[i18n.defaultLocale] || value['no'];
+    return value[locale] || value[i18n.defaultLocale] || value['no'] || value['nn'] || value['en'];
```

Comment:
kan vel ta med nb også
```suggestion
    return value[locale] || value[i18n.defaultLocale] || value['nb'] || value['no'] || value['nn'] || value['en'];
```

Reply:
`i18n.defaultLocale` er `nb` :)